docs/DICTIONARIES.md

```diff
@@ -10,10 +10,17 @@ This project uses `dictionaries` to unify props between different components.
 ## List of dictionaries
```

Comment:
This comment is meant for `/DICTIONARIES.md` in the top level: it's an empty file (that's why I'm commenting here) which probably should be deleted, is that right?